cmd/balance-worker/main.go

```diff
@@ -211,14 +210,42 @@ func main() {
 	}
 
 	// Dependencies: postgresql
-	pgClients, err := initPGClients(conf.Postgres)
+
+	// Initialize Postgres driver
+	var postgresDriver *pgdriver.Driver
```

Comment:
Not entirely sure why this variable has to be initialized.

Reply:
I do not think ther eis anything wrong with preinitializing the variable. Sometimes it makes the code more readable, but I have no strong prefenrence so I'll update this to remove the initialization.